### PR TITLE
SAK-43031 Assignments assignment with default grading and points

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 1]
```diff
@@ -11572,8 +11572,22 @@ private void initializeAssignment(SessionState state) {
 
         state.setAttribute(NEW_ASSIGNMENT_SECTION, "001");
         state.setAttribute(NEW_ASSIGNMENT_SUBMISSION_TYPE, Assignment.SubmissionType.TEXT_AND_ATTACHMENT_ASSIGNMENT_SUBMISSION.ordinal());
-        state.setAttribute(NEW_ASSIGNMENT_GRADE_TYPE, UNGRADED_GRADE_TYPE.ordinal());
-        state.setAttribute(NEW_ASSIGNMENT_GRADE_POINTS, "");
+        Boolean withGradesConfig = serverConfigurationService.getBoolean("assignment.grade.default", Boolean.TRUE);
+        state.setAttribute(WITH_GRADES, withGradesConfig);
+        if (withGradesConfig) {
+            state.setAttribute(NEW_ASSIGNMENT_GRADE_TYPE, SCORE_GRADE_TYPE.ordinal());
+            String defaultPointsConfig = serverConfigurationService.getString("assignment.points.default", "10");
```
Comment (ottenhoff) [inline assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:11579]: this doesn't seem to be what people want as points are too opinionated of a topic

Comment (ern) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:11579]: Agreed I didn't know what is the most ideal, maybe the best is to leave it unset?

Comment (ottenhoff) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:11579]: best to leave it unset

Comment (ern) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: updated

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**